cmd/controller/main.go

```diff
@@ -101,6 +101,7 @@ func main() {
 	}
 
 	options := reconciler.Options{
+		Context:                 ctx,
```

Comment:
Storing contexts in structs is an antipattern to avoid if possible: https://go.dev/blog/context-and-structs. 

Could instead the context be passed into each controller's constructor?

Reply:
yea that's def better